content/v2.0/reference/flux/stdlib/experimental/http/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Flux HTTP package
+list_title: HTTP package
+description: >
+  The Flux Experimental HTTP package provides experimental functions for transferring
+  data using the HTTP protocol.
+  Import the `experimental/http` package.
+menu:
+  v2_0_ref:
+    name: HTTP
+    identifier: HTTP-exp
+    parent: Experimental
+weight: 201
+v2.0/tags: [functions, http, package]
+---
+
+The Flux Experimental HTTP package provides experimental functions for transferring
```

Comment:
Consider: "The Flux Experimental HTTP package provides ~experimental~ functions for transferring"